src/csl/rendering/mod.rs

```diff
@@ -202,7 +202,8 @@ impl<'a, 'b> ResolvedTextTarget<'a, 'b> {
                 ctx.style.get_macro(name).map(ResolvedTextTarget::Macro)
             }
             TextTarget::Term { term, form, plural } => {
-                ctx.term(*term, *form, *plural).map(ResolvedTextTarget::Term)
+                ctx.term(*term, *form, *plural, ctx.instance.locale)
+                    .map(ResolvedTextTarget::Term)
```

Comment:
Why the formatting change? Is your `rustfmt` up to date?

Reply:
I don't use nightly build, but stable one is up to date:
<img width="1188" alt="image" src="https://github.com/typst/hayagriva/assets/56316351/a6a61152-fc75-4add-89f5-43747e6f9209">